### DISCUSSION TITLE
Enhancement: Customizable Dashboard

### DISCUSSION THREAD

**Discussion by DrMxrcy** [IceWhaleTech/CasaOS#177] [Ideas] [answered]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated having to use another Dashboard, so having the option to add links on the CasaOS Dashboard would be a great option

**Describe the solution you'd like**
Here are Some Features

- **Main Request:** I would like to be able to link the other services I had to manually install, add websites that are important to me, etc. The Ability to add custom thing to the Homescreen

- **URLs:** I would Like to Also Have Links to the Local URL and Public URL If One

- **Integrations:** A way to Integrate with apps like Sonarr, Radarr, Plex to have stats on the homepage, grabbed from the Respective APIs

- **Widgets/Modules**: A way to move the widgets and modules around more, Plus a widget/module store for things like Git Issues Trackers or other things like that.

**Describe alternatives you've considered**
I have been using [Dashy ](https://github.com/Lissy93/dashy) at the moment

**Comment by Lauren-ED209** [comment]
What's your favorite part of Dashy?

**Reply by DrMxrcy** [reply]
My favorite part is the overall customizability. The ability to set custom Icons, status checks, categories are all great features. Other dashboards like Heimdall also have integrations with popular apps like Sonnar, Radarr, Etc

**Reply by BryceLim** [reply]
It would also be nice if we could reorganize the apps.... the web GUI could be better optimized to fill the empty space at the side to allow all the apps to be fitted into one page without scrolling.

**Reply by BryceLim** [reply]
Edit: Just realised there was an update to reorganize the apps!!! Nice.

**Reply by Lauren-ED209** [reply]
Yes, we are actively updating. Storage and App center are two very basic and core sectors. 
Look forward to more suggestions. ;)

**Reply by DrMxrcy** [reply]
> Yes, we are actively updating. Storage and App center are two very basic and core sectors. 
> 
> Look forward to more suggestions. ;)

Glad my suggestions could help! Lol

**Reply by DrMxrcy** [reply]
@Lauren-ED209, I updated my Request to hopefully outline some of the features better :)

**Comment by JohnGuan** [comment, marked as answer]
The ideas you mentioned in Main Request and URLs are already in the development iteration backlog. We will discuss in the product meeting to decide the exact shape and development plan.

Integrations and widgets are good ideas that the team is still learning and will be written into a specific backlog.

I will update here if there are any updates.

**Reply by Dev-MihirParmar** [reply]
Any updates
